packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -51,6 +51,11 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.CosmosMainn
     return ChainAdapterDisplayName.Cosmos
   }
 
+  getName() {
+    const enumIndex = Object.values(ChainAdapterDisplayName).indexOf(ChainAdapterDisplayName.Cosmos)
```

Comment:
Couldn't justify bringing a new enum in when the enum keys of `ChainAdapterDisplayName` is all we actually need